pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -208,6 +208,9 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 		doc.Dequeues = 0
 	}
 
+	// TODO: Remove this once 23-04-01 API release is complete.
+	determineOutboundType(ctx, doc, subscription)
```

Comment:
should only be in `if isCreate` 

Otherwise clusters which are already created an run an update might accidentally get the UDR outbound type.